worked-solutions/01_penguin_classification_solutions.ipynb

```diff
@@ -520,17 +558,20 @@
     "\n",
     "- Before we jump in and write these loops, we must first choose an activation function to apply to the model's outputs.\n",
     "  - Here we are going to use the softmax activation function: see [the PyTorch docs](https://pytorch.org/docs/stable/generated/torch.nn.Softmax.html).\n",
-    "  - For those of you who've studied physics, you may be remininded of the partition function in thermodynamics.\n",
```

Comment:
I quite like this comment and would like it to be kept.

Reply:
Corrected